src/extension/instrument.ts

```diff
@@ -87,3 +95,10 @@ function getPageScriptCode() {
   };
   return `(${closureFn.toString()})();`;
 }
+
+function getPageUnmaskBodyCode() {
+  let closureFn = function () {
+      document.body.setAttribute("data-clarity-unmask", "true");
```

Comment:
>"data-clarity-unmask" [](start = 33, length = 21)

Should we not create an option to control this in the popup menu?

Reply:
I did - 'CSS Rules'